reactioncommerce/reaction
# refactor: remove getGraphQLContextInMeteorMethod from sitemaps-generator plugin

Resolves parts of #5529 
Impact: **minor**  
Type: **refactor**

## Issue
`generate-sitemaps` still use `getGraphQLContextInMeteorMethod`

## Solution
Move the `generate-sitemaps-job` into a startup function, and pass `context` from the startup function instead of using `getGraphQLContextInMeteorMethod()`

## Breaking changes
None

## Testing
1. Start the app
1. see that the `generate-sitemaps-job` ran without error on startup
